### PR TITLE
Fixes and improvements

### PR DESCRIPTION
**Pull request by DanielJohnHarty**
#### Which issue(s) this PR fixes:

Fixes #

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: inventory/group_vars/all.yml [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
-  - ['192.168.32.14', master-02, tdp, 3, 3072]
-  - ['192.168.32.15', master-03, tdp, 3, 3072]
-  - ['192.168.32.16', edge-01, tdp, 2, 2048]
+  - ['192.168.56.10', worker-01, tdp, 2, 2048]
+  - ['192.168.56.11', worker-02, tdp, 2, 2048]
+  - ['192.168.56.12', worker-03, tdp, 2, 2048]
+  - ['192.168.56.13', master-01, tdp, 3, 3072]
+  - ['192.168.56.14', master-02, tdp, 3, 3072]
+  - ['192.168.56.15', master-03, tdp, 3, 3072]
+  - ['192.168.56.16', edge-01, tdp, 2, 2048]
```
Comment (leopaul36) [inline inventory/group_vars/all.yml:11]: Why did you change the subnet ?

File: inventory/group_vars/all.yml [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
 
 # TDP
 realm: REALM.TDP
-num_vagrant_processes: 7
+num_vagrant_processes: 4
```
Comment (leopaul36) [inline inventory/group_vars/all.yml:15]: Why should we change this ?

Comment (DanielJohnHarty) [reply inventory/group_vars/all.yml:15]: The VMs regularly remain off with the process count at 7 on my machine. This error isn't present when using just 4. The getting started cluster requirements are based on my machine so I think it makes sense to reduce this.

File: README.md [hunk 3]
```diff
@@ -255,7 +255,7 @@ Deploys HBase masters, regionservers, rest and clients to the `[hbase_master]`,
 ansible-playbook deploy-hbase.yml
 ```
 
-As `tdp_user` on an `[hbase_client]` host, obtain a Kerberos TGT with the command `kinit -kt /home/tdp_user/.keytabs/tdp_user.principal.keytab tdp_user@REALM.TDP` and access the HBase shell with the command `/opt/tdp/hbase/bin/hbase --config /etc/hbase/conf shell`.
+As `tdp_user` on an `[hbase_client]` host, obtain a Kerberos TGT with the command `kinit -kt ~/tdp_user.keytabtdp_user@REALM.TDP` and access the HBase shell with the command `/opt/tdp/hbase/bin/hbase --config /etc/hbase/conf shell`.
```
Comment (leopaul36) [inline README.md:258]: Missing space here

File: deploy-hdfs.yml [hunk 4]
```diff
@@ -1,3 +1,6 @@
 ---
+- name: Deploy jmx exporters
+  import_playbook: ansible_roles/collections/ansible_collections/tosit/tdp/playbooks/exporter.yml
+
```
Comment (leopaul36) [inline deploy-hdfs.yml:4]: Please remove this for now on, I'll add it in a next PR

**Comment by leopaul36** [comment]
Messed up the merge, the good PR is #49
